### PR TITLE
Tone down the strong beliefs of why this is needed

### PR DESCRIPTION
**Pull request by paulehoffman**

### CODE REVIEW

File: draft-thomson-gendispatch-no-expiry.md [hunk 1]
```diff
@@ -62,20 +62,18 @@ After this expiration date, some systems might display an Internet-Draft
...
-more recently, expiration only caused the document to be hidden from certain
+more recently, expiration only causes the document to be hidden from certain
 views or searches.
 
 Copies of expired drafts are retained and can be obtained using other services.
-Expired drafts are routinely cited and referenced.  Published RFCs routinely
-include informative references to drafts, which then usually expire.
-Many IANA registries refer to expired drafts.
-Thus, statements about it being inappropriate to cite drafts are incorrect and
+Expired drafts are routinely cited and referenced in various contexts, such as in IANA registries and informational references in RFCs.
```
Comment (martinthomson) [inline draft-thomson-gendispatch-no-expiry.md]:
```suggestion
Expired drafts are routinely cited and referenced in various contexts,
such as in IANA registries, academic papers, and informational references in RFCs.
```

**Review by martinthomson — APPROVED** [review]
Seems reasonable.

**Comment by elear** [comment]
If I may suggest one change, where it reads:

... "are actually used"; change to "may be used.”

**Comment by paulehoffman** [comment]
Fixed. With that, are you OK with this PR?

**Comment by elear** [comment]
Yes!  I still want to cover other ground outside of the Intro, but this is a good improvement.

**Pull request merged**